glslc/src/main.cc

```diff
@@ -97,11 +98,18 @@ bool GetOptionArgument(int argc, char** argv, int* index,
 }  // anonymous namespace
 
 int main(int argc, char** argv) {
-  std::vector<std::pair<std::string, EShLanguage>> input_files;
-  EShLanguage force_shader_stage = EShLangCount;
+  std::vector<std::pair<std::string, shaderc_shader_kind>> input_files;
+  shaderc_shader_kind forced_shader_stage = shaderc_glsl_infer_from_source;
```

Comment:
I wonder if there's a better name for `forced_shader_stage`.  It's just the value of the latest `-fshader-stage` flag, right?  So, perhaps something like `current_fshader_stage` might be better?